From 34d5779d41a902be21cef3fc3e70d8f6881ca05c Mon Sep 17 00:00:00 2001
From: Jakub Jirutka <jakub@jirutka.cz>
Date: Mon, 27 Sep 2021 01:00:43 +0200
Subject: [PATCH] [rofi-sensible-terminal] Use /bin/sh instead of /usr/bin/env bash
Patch-Source: https://github.com/davatorium/rofi/pull/1478

This script is fully POSIX-sh compatible, i.e. it's portable and can be
run in any shell, not just bash. It's incorrect to assume that bash is
available on every *nix system (it's not), but /bin/sh is.
---
 script/rofi-sensible-terminal | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/script/rofi-sensible-terminal b/script/rofi-sensible-terminal
index 112e1d14e..e2b6cc4b9 100755
--- a/script/rofi-sensible-terminal
+++ b/script/rofi-sensible-terminal
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/sh
 #
 # This code is released in public domain by Han Boetes <han@mijncomputer.nl>
 # Updated by Dave Davenport <qball@gmpclient.org>
